src/main/java/Course.java

```diff
@@ -0,0 +1,68 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Course implements Comparable<Course>{
```

Comment:
overriding equals and hashcode methods could help with storing users in collections and especially to avoid having the same user twice in the same collection (e.g. [here ](https://github.com/ElenaBezro/StudentManagementSystem/pull/1/files#diff-c468baa8c7536e13b0624e5b73c72fefed8409183ce9a249c8ba432acf3ff761R17))